meta/core.md

```diff
@@ -27,6 +27,34 @@ To be able to distinguish Dimensions that are defined inside a Cube from Dimensi
 
 </aside>
 
+### meta:Limit {#Limit}
+
+A type of [annotation](#annotation) which can be used to express a limit or target value for a dimension.
+
+<aside class='example' title='Dimension with a target'>
+
+```turtle
+<cube/shape> sh:property [
+  a cube:KeyDimension ;  
+  sh:path ex:year ;
+] , [
+  a cube:MeasureDimension ;
+  meta:annotation [
+    a meta:Limit ;
+    schema:value 95 ;
```

Comment:
Here, this part, `schema:value` is too generic. Do we mean a lower or an upper limit?

We can avoid this confusion in at least three ways I could think of:
1. Introduce two new classes `meta:UpperLimit` and `meta:LowerLimit` 
2. Add some other specification, like an additional `meta:limitType "upper"` or something like that
3. Be clear, on what we mean, by replacing `schema:value` by something more precise, like a `schema:maxValue` for an upper limit or a `schema:minValue` for a lower limit.

I would advocate together with Thomas for option three, as it reuses `schema:maxValue`/`schema:minValue` instead of introducing new things.

Reply:
Ok, I think I understood now.

Let me clarify one more point. Would it ever make sense to have both `schema:minValue` and `schema:maxValue` on a limit. From what you write, I guess that is pointless?

Reply:
Depends on how you want to look at it.

It could be, that something is supposed to follow an upper and a lower limit. One could then add a `schema:minValue` as well as a `schema:maxValue` to the same `meta:Limit`. Or be more precise and add two `meta:Limit`.

From a semantic's point it might be clearer to add two `meta:Limit` in such a case. So yes, there's not really a point to have both predicates for one `meta:Limit`. Or would you like to disagree @Rdataflow ?

Reply:
I just imagined the case of a target range, being specified by means of lower and upper limit. Yes, for such cases it could make sense to have both at the some node ;-)

Reply:
Two thoughts:
1. It could make sense to have min and max, together, so to allow multiple "limits" with corresponding upper and lower limit. (e.g. Target Band 2025, Target Band 2026) so I propose to keep connected min / max together.
2. It should be possible, to give a limit annotation a title, and probably also a sub-title/description. There might be multiple limit annotations, and earlier or later even a possibility to choose one, or multiple to show in an interface (e.g. visualize).

Reply:
Alright, you got me conviced. I was steering more towards thinking of target ranges as "two limits in a trenchcoat" and thus wanting to split them up. But that would just be more "wordy". So let's keep the possibility.

As for your second thought @l00mi , I agree. a `dcterms:title` would make sense.

Reply:
In principle, any property is allowed. The shapes aren't closed. Do you think it's worth mentioning that explicitly in the spec?

Reply:
https://github.com/zazuko/cube-link/pull/206
@Kronmar-Bafu @Rdataflow